nss-day-cohort-48/giffygram-jumpstation
# Nrh new posts

#### Changes Made

1. Modified file PostList to include rendering of submitted posts.
2. Added logic to send posts to API and fetch them when rendering page.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout nrh-new-posts
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   Verify once logging in that you can create a new post and once it is submitted that It renders the page with your post visible.
4. View code file.
   Look at the code and verify it is free of spaghetti.
